server/a_socials/settings/base.py

```diff
@@ -40,58 +40,66 @@
 # Application definition
 
 INSTALLED_APPS = [
-  'django.contrib.admin',
-  'django.contrib.auth',
-  'django.contrib.contenttypes',
-  'django.contrib.sessions',
-  'django.contrib.messages',
-  'django.contrib.staticfiles',
-  'webpack_loader',
-  'api',
-  'rest_framework',
-  'django_filters',
-  'corsheaders',
-  'graphene_django',
-  'graphql_schemas'
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'webpack_loader',
+    'api',
+    'rest_framework',
+    'django_filters',
+    'corsheaders',
+    'graphene_django',
+    'graphql_schemas',
+    'django_crontab',
 ]
 
+CRONJOBS = [
+    ('* * * * *', 'graphql_schemas.utils.cron_jobs.check_and_notify_members',
+     '>> ~/Andela-Socials/file.log')
```

Comment:
Do we need a `file.log` file?

Reply:
@OsayamenOsaretin I don't think so this was just for debug purposes, I will remove it.